atomicapp/nulecule/main.py

```diff
@@ -90,6 +90,16 @@ def __init__(self, app_spec, destination=None, answers_file=None):
             if os.path.isfile(os.path.join(self.app_path, ANSWERS_FILE)):
                 self.answers_file = os.path.join(self.app_path, ANSWERS_FILE)
 
+        # TODO: put this in a better place in the future.
```

Comment:
@dustymabe Okay, I'm okay for merging, but this needs to be changed in the future as I feel as though it's not suppose to belong there.